lr_face/data.py

```diff
@@ -101,6 +116,20 @@ def get_image(
             res = res / 255
         return res
 
+    @property
+    def quality_score(self):
+        """ returns a 'quality score', as the average of the top ten score
```

Comment:
binned average?

Reply:
ja maar dat is weer verwarrend omdat we soort van maar 1 bin nemen..?